### DISCUSSION TITLE
app/How-to-install-Hiddify-app/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#3936] [Content Help کمک به اصلاح محتوا]
# app/How-to-install-Hiddify-app/

The proxy manager platform.

https://hiddify.com/app/How-to-install-Hiddify-app/

**Comment by Enifax** [comment]
How to install on IOS ?  I can't find it in the applestore

**Comment by fadeshun1** [comment]
How to find the installation package on MAC? The download link you provided is invalid! !
